### PR TITLE
[feat] Add fastai integration

### PR DESCRIPTION
**Pull request by tmynn**
Added callback class for fastai, and an example

### CODE REVIEW

**Comment by gorarakelyan** [comment]
resolves #462

**Review by mihran113 — APPROVED** [review]
looks great!

File: aim/sdk/adapters/fastai.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            self.setup()
+        return self._run
+
+    def setup(self):
+        if not self._run:
+            if self._run_hash:
+                self._run = Run(
+                    self._run_hash,
+                    repo=self.repo,
+                    system_tracking_interval=self.system_tracking_interval,
```
Comment (alberttorosyan) [inline aim/sdk/adapters/fastai.py:43]: seems `log_system_params` is missing here

File: aim/sdk/adapters/fastai.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                self._run = Run(
+                    repo=self.repo,
+                    experiment=self.experiment_name,
+                    system_tracking_interval=self.system_tracking_interval,
+                    log_system_params=self.log_system_params,
+                )
+                self._run_hash = self._run.hash
+
+        # Log config parameters
+        configs_log = self.gather_args()
```
Comment (alberttorosyan) [inline aim/sdk/adapters/fastai.py:55]: what about moving the arguments gathering/setting run h-params directly into `before_fit`?
`setup` is meant to create/re-create the inner Run object.

**Review by alberttorosyan — COMMENTED** [review]
looks good! left a couple of comments.
lets also add documentation with this PR

File: aim/sdk/adapters/fastai.py (hunk 2 above)
Comment (tmynn) [reply aim/sdk/adapters/fastai.py:55]: Should we gather the arguments twice? Once in `experiment` and one in `before_fit`.

**Review by alberttorosyan — APPROVED** [review]

**Pull request merged**